mlem/core/metadata.py

```diff
@@ -142,6 +145,7 @@ def load_meta(
     follow_links: bool = True,
     load_value: bool = False,
     fs: Optional[AbstractFileSystem] = None,
+    batch: Optional[int] = None,
```

Comment:
`load_meta` should not have batch arg: you are only loading metadata. `load_value` here is for convenience. If you need batching, you should set `load_value=False` and then call `read_batch` on `DatasetMeta` directly